module-services/service-cellular/call/CellularCall.cpp

```diff
@@ -186,6 +186,24 @@ namespace CellularCall
         cpuSentinel = std::move(sentinel);
     }
 
+    void Call::handleCallAudioEventRequest(cellular::CallAudioEventRequest::EventType event)
+    {
+        switch (event) {
+        case cellular::CallAudioEventRequest::EventType::Mute:
+            audio.muteCall();
+            break;
+        case cellular::CallAudioEventRequest::EventType::Unmute:
+            audio.unmouteCall();
+            break;
+        case cellular::CallAudioEventRequest::EventType::LoudspeakerOn:
+            audio.setLaudSpeakerOn();
+            break;
+        case cellular::CallAudioEventRequest::EventType::LoudSpeakerOff:
+            audio.setLaudSpeakerOff();
+            break;
```

Comment:
```suggestion
            audio.setLoudspeakerOn();
            break;
        case cellular::CallAudioEventRequest::EventType::LoudspeakerOff:
            audio.setLoudspeakerOff();
            break;
```
Loudspeaker is one word, so it shouldn't be capitalized this way: LoudSpeaker